crates/oxc_parser/src/lib.rs

```diff
@@ -696,6 +696,23 @@ mod test {
         assert_eq!(ret.errors.first().unwrap().to_string(), "Source length exceeds 4 GiB limit");
     }
 
+    #[test]
```

Comment:
This file is for integration tests; parser tests should go into https://github.com/oxc-project/oxc/tree/main/tasks/coverage/misc/pass

Reply:
I think this fix can only be verified by examining the ast. It can't be done in there, can it?